server/src/main/java/org/opensearch/index/mapper/FieldMapper.java

```diff
@@ -278,23 +281,27 @@ public void parse(ParseContext context) throws IOException {
                     valuePreview = complexValue.toString();
                 }
             } catch (Exception innerException) {
+                if (!ignore_malformed) {
```

Comment:
We should have a spotless check for this, but for now we just ask contributors use explicit logic checks instead of negations.

```suggestion
                if (ignore_malformed == false) {
```

Reply:
Sure!